### PR TITLE
I finished the class exercises and the homework too :-) 

### PR DESCRIPTION
**Pull request by ghost**
I finished the class exercises and the homework too :-) ..  I look forward to receiving your feedback.

Thanks.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
```
Comment (bitsagarob) [inline .eslintrc.js]: this file should not be committed

File: .eslintrc.json [hunk 2]
```diff
@@ -1,12 +1,19 @@
 {
```
Comment (bitsagarob) [inline .eslintrc.json]: file should not be committed

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,188 @@
...
+    'angels_and_demons'
+  ]
+  // Funtions
+  function createElement(typeelment) {
+    return document.createElement(typeelment)
+  }
+  function addToParent(parent, child) {
+    return parent.appendChild(child)
+  }
+  function addArrayAndObjectsToList(array, object, img) {
```
Comment (bitsagarob) [inline Week1/homework/app.js]: try to split this function into separate smaller reusable functions

**Review by bitsagarob — APPROVED** [review]

**Comment by ghost** [comment]
Files .eslintrc.json and .eslintrc.js were updated to be  omited the commit and  I split the addArrayAndObjectsToList in two functions small ones: addObjLibraryToList and  addObjImgToList.

**Pull request closed without merging**